.github/workflows/publish_to_test.yaml

```diff
@@ -14,18 +14,26 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
-    - name: Setup node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: 16.4
+      - name: Setup node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.4
 
-    - name: Build website
-      run: npm install gatsby-cli && npm run build
+      - name: Build website
+        run: npm install gatsby-cli && npm run build
 
-    - name: Sync files to S3
-      run: aws s3 sync ./public/ s3://$AWS_BUCKET_NAME --acl public-read --delete --cache-control max-age=604800
+      - name: Sync files to S3
+        run: aws s3 sync ./public/ s3://$AWS_BUCKET_NAME --acl public-read --delete --cache-control max-age=604800
 
-    - name: Notify CloudFront about the changes
-      run: aws cloudfront create-invalidation --distribution-id ${AWS_DISTRIBUTION_ID} --paths "/*"
+      - name: Notify CloudFront about the changes
+        run: aws cloudfront create-invalidation --distribution-id ${AWS_DISTRIBUTION_ID} --paths "/*"
+  broken-link-check:
```

Comment:
this task needs to be in another `test.yaml` file, so everytime when we merge it the action will be triggered. like here: https://github.com/alephium/alephium-js/blob/master/.github/workflows/test.yml

Reply:
Thank you @polarker, I've updated the PR!